std/uni.d

```diff
@@ -7578,7 +7578,7 @@ public:
         }
     }
 
-    ~this() pure @nogc nothrow
+    ~this() pure @nogc nothrow @trusted
     {
         import core.memory : pureFree;
```

Comment:
Go with `dispose`, instead:

```d
import std.experimental.allocator.mallocator : Mallocator;
import std.experimental.allocator : makeArray, dispose;
alias alloc = Mallocator.instance;
auto p = alloc.makeArray!ubyte(20);
alloc.dispose(p);
```
https://run.dlang.io/is/t2IMq2

but it needs to be fixed first (https://issues.dlang.org/show_bug.cgi?id=18347)